xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/liveDataSource.js

```diff
@@ -23,6 +23,8 @@ define('xwiki-livedata-source', ['module', 'jquery'], function(module, $) {
 
   var baseURL = module.config().contextPath + '/rest/liveData/sources/';
 
+  var entriesRequest = null;
```

Comment:
> Is it fine to use let here where all the other declarations use var?

We should move to ``let`` / ``const`` progressively. So whenever we touch code that use ``var`` we should refactor it.

> (I also set null explicitly because I like that it's explicit but I don't care much either way - let vs var does not bother me too much, though I indeed prefer let usually)

I'm fine with this.